queries/orderbook/prod_batch_rewards.sql

```diff
@@ -304,7 +304,10 @@ order_protocol_fee_prices AS (
         END AS network_fee_correction,
         opf.sell_token as network_fee_token,
         ap_surplus.price / pow(10, 18) as surplus_token_native_price,
-        ap_protocol.price / pow(10, 18) as protocol_fee_token_native_price,
+        CASE
+            WHEN opf.order_uid = '\x942e55be89314c5e799a12b487387a1221e5671bd17222a1bc69cb2b7728a4eea53a13a80d72a855481de5211e7654fabdfe352666e32aec' and opf.auction_id = 9269949 then 1.9088477506249592e-5
```

Comment:
This is fine now.

For future changes, we could even go with
```suggestion
            WHEN opf.protocol_fee_token = '\xad038eb671c44b853887a7e32528fab35dc5d710' and opf.auction_id = 9269949 then 1.9088477506249592e-5
```
since that makes clear that we are changing the price of a token for some auction.